.drone.star

```diff
@@ -265,6 +263,7 @@ def gui_tests(squish_parameters = "", depends_on = []):
             "SQUISH_PARAMETERS": squish_parameters,
             "STACKTRACE_FILE": STACKTRACE_FILE,
         },
+        "security_opt": "seccomp=unconfined",
```

Comment:
What problem do you want to solve by removing the confinement, i.e., vastly decreasing the security of the CI pipeline?

Reply:
Yeah, I was just trying check if drone would support that or not. I would not implement that anyway but that was the only workaround to make xfce4 of latest squish image work on ubuntu20.04 host (on my local system)